From aa4c15388c4362546cd7b1018adc641a02f50571 Mon Sep 17 00:00:00 2001
From: Rohit Athavale <rathaval@xilinx.com>
Date: Fri, 23 Feb 2018 11:15:08 -0800
Subject: [PATCH 0307/1852] Documentation: media: Update dt-bindings doc for
 10-bit color depth in vpss-csc

commit 71e8a0a320b20507e2bb244a118b1f3f753523ea from
https://github.com/Xilinx/linux-xlnx.git

Add documentation stating support for 10 bit depth for vpss-csc.

Signed-off-by: Rohit Athavale <rathaval@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt
index 608be04391ae..e1cde4a0a42d 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt
@@ -22,7 +22,7 @@ Required port properties:
   for input port (0) and XVIP_VF_RBG, XVIP_VF_YUV_444 or XVIP_VF_YUV_422
   for output port (1). See <dt-bindings/media/xilinx-vip.h> for more details.
 
-- xlnx,video-width: Video width as defined in video.txt
+- xlnx,video-width: Video width as defined in video.txt. Must be either 8 or 10.
 
 Example:
 	csc_1:csc@a0040000 {
-- 
2.31.1

